eng/testing/tests.mobile.targets

```diff
@@ -157,28 +157,40 @@
       </WasmSatelliteAssemblies>
     </ItemGroup>
     <ItemGroup>
-      <AssemblySearchPaths Include="$(PublishDir)"/>
+      <Assemblies Include="$(PublishDir)\*.dll"/>
       <WasmFilesToIncludeInFileSystem Include="@(ContentWithTargetPath)" />
       <WasmFilesToIncludeInFileSystem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.BuildReference)' == 'true' and !$([System.String]::new('%(ReferenceCopyLocalPaths.Identity)').EndsWith('.resources.dll'))" />
       <WasmFilesToIncludeInFileSystem Include="@(WasmSatelliteAssemblies)" TargetPath="%(WasmSatelliteAssemblies.CultureName)\%(WasmSatelliteAssemblies.Filename)%(WasmSatelliteAssemblies.Extension)" />
       <!-- Include files specified by test projects from publish dir -->
-      <WasmFilesToIncludeInFileSystem Include="@(WasmFilesToIncludeFromPublishDir -> '$(PublishDir)%(Identity)')" />
-      <ExtraAssemblies Include="$(PublishDir)$(AssemblyName).dll" />
+      <WasmFilesToIncludeInFileSystem Include="@(WasmFilesToIncludeFromPublishDir -> '$(PublishDir)\..\%(Identity)')" />
+      <Assemblies Include="$(PublishDir)\..\$(AssemblyName).dll" />
       <!-- We need these facades for some tests. -->
-      <ExtraAssemblies Include="$(PublishDir)mscorlib.dll" />
-      <ExtraAssemblies Include="$(PublishDir)System.Drawing.dll" />
+      <Assemblies Include="$(PublishDir)\..\mscorlib.dll" />
+      <Assemblies Include="$(PublishDir)\..\System.Drawing.dll" />
+      <!-- WasmTestRunner referenced assemblies -->
```

Comment:
You shouldn't need to explicitly call these out.  Would recommend removing.

Reply:
There were errors regarding missing assemblies that `WasmTestRunner.dll` depended on because of the `<PublishTrimmed>true</PublishTrimmed>` addition. So I'll revert those changes long with removing `<PublishTrimmed>true</PublishTrimmed>`.

Reply:
Yeah, we won't need that for the tests.  Too much pain.